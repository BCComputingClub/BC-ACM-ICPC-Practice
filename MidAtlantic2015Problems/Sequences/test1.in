1 2 -1 4
2 4 8 -1
7 8 -1 21
5 -1 11 14
-1 2 4 6
-1 1006 2006 3006
-1 20 120 220
-1 27 81 243
-1 27 81 244
-1 3 6 12
-1 1 2 3
-1 37 185 925
-1 9970 9990 10000
6 -1 22 30
6 -1 22 31
90 -1 360 720
9997 -1 9999 10000
9 -1 36 108
12 17 -1 27
13 17 -1 27
7 14 -1 56
7 21 -1 56
9700 9800 -1 10000
25 27 29 -1
132 232 332 -1
26 27 29 -1
32 132 232 -1
450 900 1800 -1
450 900 2700 -1
1 100 10000 -1
127 256 -1 1024
-1 -1 -1 -1
