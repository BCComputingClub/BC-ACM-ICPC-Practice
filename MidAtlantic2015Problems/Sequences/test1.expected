3
16
-1
8
-1
6
-1
9
-1
-1
-1
-1
-1
14
-1
180
9998
-1
22
-1
28
-1
9900
31
432
-1
332
3600
-1
1000000
-1
